.github/workflows/docker.yaml

```
@@ -113,6 +113,8 @@ jobs:
         image: ${{ needs.build_image.outputs.image_tag }}
         ports:
           - 7860:7860
+        env:          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)